packages/core/ios/Sources/DdSdk.swift

```diff
@@ -292,7 +292,7 @@ class RNDdSdk: NSObject {
             Global.rum.updatePerformanceMetric(metric: .jsFrameTimeSeconds, value: frameTime)
         }
         if (frameTime > self.jsLongTaskThresholdInSeconds) {
-            // TODO: report long task
+            Global.rum._internal?.addLongTask(at: Date(), duration: frameTime, attributes: ["long_task.target": "javascript"])
```

Comment:
I will make a change in the iOS SDK so that we don't provide the date, but rather use the RUM time provider on the iOS SDK side when reporting long tasks